project.json

```diff
@@ -1,6 +1,6 @@
 {
-  "serviceRole": "",
-  "serviceType": "",
+  "serviceRole": "exoservice-go",
```

Comment:
Thoughts on making this default value `ping-service` since the default for this is essentially just a service that responds to pings?